airflow/logging/airflow_task_handler.py

```diff
@@ -0,0 +1,244 @@
+# -*- coding: utf-8 -*-
```

Comment:
please use `airflow.utils.log.TaskFileHandler` this properly identifies what is does. `airflow_task_handler` is ambiguous (file/s2/gcs?)